configuration/pih/htmlforms/section-delivery.xml

```diff
@@ -208,6 +208,8 @@
                 var hasValue = function(element) {
                     return jq(element).find('.baby-name input').val();
                 };
+                restrictInputOnlyNumber("head-circumference ")
```

Comment:
Are you sure this is necessary?  Since the head circumference concept is already a numeric data type (unlike the receipt number concept which was a text datatype), I'd think that the obs input would restrict to numbers by default.  Do you really need to add this in?

Reply:
it is not necessary as you say. I was wondering that.